check-digit/src/main/scala/hmda/uli/HmdaUli.scala

```diff
@@ -26,5 +26,5 @@ object HmdaUli extends App {
   implicit val system = ActorSystem("hmda-uli")
 
   system.actorOf(HmdaUliApi.props(), "hmda-uli-api")
-  system.actorOf(HmdaUliGrpc.props(), name = "hmda-uli-grpc")
+//  system.actorOf(HmdaUliGrpc.props(), name = "hmda-uli-grpc")
```

Comment:
If you're removing GRPC you can go ahead remove this commented out code. Doesn't look like you're leaving any other references to GRPC in place for roll back.

Reply:
New commits fix the review comments.